src/features/commands/book/book.js

```diff
@@ -0,0 +1,64 @@
+const {
```

Comment:
We should have a better naming convention for the files. commands/book/book doesn't give too much info. Maybe we should have something like `commands/book/book-business`, to make clear what this is file is for when we have more inside this folder.